setup.py

```diff
@@ -461,6 +461,21 @@ def create_or_sync_database(configure):
     else:
         call(u'Migrate DB:', [u'env/bin/python', u'manage.py', u'migrate'])
 
+def load_redirects(configure):
+    from django.contrib.redirects.models import Redirect
+    from django.contrib.sites.models import Site
+
+    Redirect.objects.all().delete()
+    site = Site.objects.get(pk=1)
+    with JsonFile(u'fixtures/redirects_redirect.json', u'r') as data:
```

Comment:
@martinmacko47 JsonFile sa doteraz nikde nepouzival. Problem je ze si po zavreti suboru automaticky upravi cely subor, indent aj poradie fieldov. Na koniec suboru neda prazdny riadok. Pridanim `'r'` mu to zakazeme.

Reply:
@viliambalaz  Ano mas pravdu, subor treba otvorit read-only, nech sa neprepisuje. Haluz, ze to nerobilo problem na tom druhom projekte.